scripts/utils/cliUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { InvalidArgumentError } from '@commander-js/extra-typings'
+
+export const testVariadicArgs = (
```

Comment:
Nitpick: are we likely to have any more variadic arguments in this repo? I'd be inclined just to use standard date validation as this seems a layer of complexity we don't need? Appreciate it is elegant though and nice reusability.